test/router/v2/integration/SynapseRouterV2.Integration.Avalanche.t.sol

```diff
@@ -0,0 +1,981 @@
+                router.getDestinationBridgeTokens(expectedTokens[i]),
+                expectedDestinationBridgeTokens[expectedTokens[i]]
+            );
+        }
+    }
+
+    function testGetOriginAmountOut_inUSDCe_outNUSD() public {
+        address tokenIn = USDC_E;
+        string memory tokenSymbol = "nUSD";
+        uint256 amountIn = getTestAmount(USDC_E);
+
+        address pool = AVAX_STABLE_POOL;
+        uint8 indexFrom = 2;
+        uint8 indexTo = 0;
+        uint256 amountOut = calculateSwap(pool, indexFrom, indexTo, amountIn);
+
+        SwapQuery memory query = router.getOriginAmountOut(tokenIn, tokenSymbol, amountIn);
+        assertEq(query.routerAdapter, address(router));
+        assertEq(query.tokenOut, NUSD);
+        assertEq(query.minAmountOut, amountOut);
+        assertEq(query.deadline, type(uint256).max);
+        assertEq(query.rawParams, getSwapParams(pool, indexFrom, indexTo));
+    }
+
+    function testGetOriginAmountOut_inNUSD_outNUSD() public {
+        address tokenIn = NUSD;
+        string memory tokenSymbol = "nUSD";
+        uint256 amountIn = getTestAmount(NUSD);
+
+        uint256 amountOut = amountIn;
+        SwapQuery memory query = router.getOriginAmountOut(tokenIn, tokenSymbol, amountIn);
+        assertEq(query.routerAdapter, address(0));
+        assertEq(query.tokenOut, NUSD);
+        assertEq(query.minAmountOut, amountOut);
+        assertEq(query.deadline, type(uint256).max);
+        assertEq(query.rawParams, bytes(""));
+    }
+
+    /// @dev UNI not supported so amount out should produce zero
+    function testGetOriginAmountOut_inUSDCe_outUNI() public {
+        address tokenIn = USDC_E;
+        string memory tokenSymbol = "UNI";
+        uint256 amountIn = getTestAmount(USDC_E);
+
+        SwapQuery memory query = router.getOriginAmountOut(tokenIn, tokenSymbol, amountIn);
+        assertEq(query.routerAdapter, address(0));
+        assertEq(query.tokenOut, address(0));
+        assertEq(query.minAmountOut, 0);
+        assertEq(query.deadline, 0);
+        assertEq(query.rawParams, bytes(""));
+    }
+
+    function testGetOriginAmountOut_inUSDCe_outUSDC() public {
+        address tokenIn = USDC_E;
+        string memory tokenSymbol = "CCTP.USDC";
+        uint256 amountIn = 990000000000; // 990K USDC
+
+        address pool = 0x9A2Dea7B81cfe3e0011D44D41c5c5142b8d9abdF; // LinkedPool
+        uint8 indexFrom = 4;
+        uint8 indexTo = 0;
+        uint256 amountOut = calculateSwap(pool, indexFrom, indexTo, amountIn);
+
+        SwapQuery memory query = router.getOriginAmountOut(tokenIn, tokenSymbol, amountIn);
+        assertEq(query.routerAdapter, address(router));
+        assertEq(query.tokenOut, USDC);
+        assertEq(query.minAmountOut, amountOut);
+        assertEq(query.deadline, type(uint256).max);
+        assertEq(query.rawParams, getSwapParams(pool, indexFrom, indexTo));
+    }
+
+    /// @dev not enough liquidity to test CCTP max bridged amount out
```

Comment:
Add a test here for `router.getOriginAmountOut(USDC, "CCTP.USDC", 10**12+1)`? Should return empty struct.

Reply:
Ah true. Good call ty